### DISCUSSION TITLE
Markor autosave question - regarding deleting all contents

### DISCUSSION THREAD

**Discussion by pvautour** [gsantner/markor#1540] [Q&A] [answered]
---
Cellphone: samsung galaxy s9 plus
Os: Android 10
App store: google playstore
---

Hello,

This may be an issue that only affects me, but iIcan reproduce it consistently so I belive it is the default behavior.

Currently, if i write text in my `QuickNote`, click on the `files` and then click on the `QuickNote` tab again, the text i have written will have been saved to the file and be visible. This is correct and expected behavior.

However, if i :

1. Erase text from `QuickNote`
2. Click the `Files` tab
3. Click the `QuickNote` tab

The text i had erased reappears as if it was never erased. 

This can be circumvented by using the following steps:

1. Erase text from `QuickNote`
2. Write any character to the file.
3. Click the `Files` tab
4. Click the `QuickNote` tab

Now, you will see that the deleted text does not reappear.

My intuition is that this would be caused by the software only autosaving on write, and not on delete. It would seem like an easy fix to just trigger the autosave in deletion cases. However, I have not looked at the code so i cannot say for sure. I may look into it someday.

Thank you for the great piece of software!

Pascal V.

**Comment by gsantner** [comment, marked as answer]
When you remove all text, autosaving is for safety reasons not triggered.  
If you really want to save a file as empty, manually press the save button in the top menu (after you deleted all text :smiley: ).

Or just leave a word.
